.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
+
+### Expected behaviour
+
+### Actual behaviour
+
+### Steps to reproduce
```

Comment:
Is an `ISSUE` only ever a bug report? We use stories in Product to cover changes, tweaks, enhancement, and so on, where these headings wouldn't apply.